.circleci/config.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           at: .
       - run:
           name: build:storybook
-          command: yarn build-storybook
+          command: yarn build:storybook
```

Comment:
`yarn build-storybook` has been updated to `yarn build:storybook` to follow the colon format. All instances have been updated 

![Screenshot 2024-01-30 at 3 40 54 PM](https://github.com/MetaMask/design-tokens/assets/8112138/43266a67-5173-4d36-a603-c1e29a0484b0)